client/blocks/taxonomy-manager/list-item.jsx

```diff
@@ -121,6 +121,7 @@ class TaxonomyManagerListItem extends Component {
 
 	render() {
 		const { canSetAsDefault, isDefault, onClick, term, translate, isJetpack } = this.props;
+		const name = this.getName();
```

Comment:
getName contains some extra logic (empty names and decode entities), I'm thinking this avoid us some duplication.